modules/ROOT/pages/6.0-release-notes-ui-and-ux-elements-and-components.adoc

```diff
@@ -0,0 +1,263 @@
+= TinyMCE 6.0 UI and UX elements and components changes
+:navtitle: TinyMCE 6.0 UI and UX elements and components changes
+:description: TinyMCE 6.0 UI and UX elements and components changes
+:keywords: releasenotes, ui, ux
+
+This section documents the improvements, changes, fixes and removals made to {productname} 6.0 UI and UX elements and components.
+
+* xref:ui-and-ux-elements-and-components[UI and UX elements and components]
+* xref:new-and-improved-ui-and-ux-elements-and-components][New and improved UI and UX elements and components]
+* xref:changed-ui-and-ux-elements-and-components[Changed UI and UX elements and componen]
+* xref:fixed-ui-and-ux-elements-and-components[Fixed UI and UX elements and components]
+* xref:removed-or-deprecated-ui-and-ux-elements-and-components[Removed or deprecated UI and UX elements and components]
+
+// tag::ui-and-ux-elements-and-components[]
+[[ui-and-ux-elements-and-components]]
+== UI and UX elements and components
+
+[[new-and-improved-ui-and-ux-elements-and-components]]
+=== New and improved UI and UX elements and components
+
+[[buttontype]]
+==== `buttonType`
+
+`buttonType` is a new property available as a basic panel component of `dialogs`.
```

Comment:
This isn't quite correct... it's a new property that can be used for buttons components in dialogs. So maybe something like this?
```suggestion
`buttonType` is a new property available for button components used in dialogs.
```